### PR TITLE
 miner: gas limit strategy, target 3141592 & def gas price 50 Shannon 

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**

### CODE REVIEW

**Comment by robotally** [comment]
:exclamation: Pull request against `master` :exclamation:

| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Mon Aug 3 09:42:59 UTC 2015_

**Comment by obscuren** [comment]
:+1:

**Comment by DannyDesert** [comment]
get ready

File: core/chain_util.go [hunk 1]
```diff
@@ -69,12 +69,27 @@ func CalcTD(block, parent *types.Block) *big.Int {
...
 func CalcGasLimit(parent *types.Block) *big.Int {
-	decay := new(big.Int).Div(parent.GasLimit(), params.GasLimitBoundDivisor)
+	// contrib = (parentGasUsed * 3 / 2) / 1024
 	contrib := new(big.Int).Mul(parent.GasUsed(), big.NewInt(3))
 	contrib = contrib.Div(contrib, big.NewInt(2))
 	contrib = contrib.Div(contrib, params.GasLimitBoundDivisor)
 
+	// decay = parentGasLimit / 1024 -1
+	decay := new(big.Int).Div(parent.GasLimit(), params.GasLimitBoundDivisor)
+	decay.Sub(decay, big.NewInt(1))
```
Comment (subtly) [inline core/chain_util.go]: Shouldn't this be on line 99?

Comment (obscuren) [inline core/chain_util.go]: Yes (this is the wrong PR btw)

**Pull request closed without merging**